project/src/main/java/com/google/sps/data/Restaurant.java

```diff
@@ -0,0 +1,81 @@
+package com.google.sps.data;
+
+import java.lang.Iterable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/** Represents a restaurant with it's relevant information. */
+public class Restaurant {
+  /** Represents a price level for the restaurant. */
+  public enum PriceLevel { LOW, MED, HIGH, UNKNOWN }
+  ;
+
+  private final String name;
+  private final String cuisineType;
+  private final String address;
+  private final Map<String, Double> latLngCoords;
+  private final double avgRating;
+  private final int numRatings;
+  private final PriceLevel priceLevel;
+  private final String id;
+  private final List<String> placeTypes;
+
+  public Restaurant(String name, String cuisineType, String address,
+      Map<String, Double> latLngCoords, double avgRating, int numRatings, int priceNum, String id) {
+    this.name = name;
+    this.cuisineType = cuisineType;
+    this.address = address;
+    this.avgRating = avgRating;
+    this.numRatings = numRatings;
+    switch (priceNum) {
+      case 0:
+      case 1:
+        priceLevel = PriceLevel.LOW;
+        break;
+      case 2:
+        priceLevel = PriceLevel.MED;
+        break;
+      case 3:
+      case 4:
+        priceLevel = PriceLevel.HIGH;
+        break;
+      default:
+        priceLevel = PriceLevel.UNKNOWN;
+        break;
+    }
```

Comment:
I feel like our `PriceLevel` enum should more closely correspond with Google's [description](https://developers.google.com/places/web-service/search):
```
0 — Free
1 — Inexpensive
2 — Moderate
3 — Expensive
4 — Very Expensive
```
Also, if we do a 1-to-1 mapping between int and enum, we can resolve the int to enum with `priceLevel = PriceLevel.values()[intval]`